settings.gradle

```diff
@@ -112,7 +112,7 @@ include ':instrumentation:netty:netty-3.8'
 include ':instrumentation:netty:netty-4.0'
 include ':instrumentation:netty:netty-4.1'
 include ':instrumentation:okhttp-3.0'
-include ':instrumentation:opentelemetry-api-0.3'
+include ':instrumentation:opentelemetry-api-0.4'
```

Comment:
Why do we want version number here? Any given version of auto-instr supports one specific version of Otel-java. This version should be present in documentation and in `dependencies.gradle`. The requirement to rename folders after every release of otel-java seems to me as a lot of trouble without any benefit.

Reply:
We're planning to support multiple versions of opentelemetry-api, so users can upgrade to the latest auto-instrumentation without having to update their manual instrumentation. For now though since we're in beta, we're just dropping support for older opentelemetry-api version when we update to newer version.

Reply:
Maybe `instrumentation:opentelemetry-api-beta` or some variation would make sense?